build/build.js

```diff
@@ -30,23 +30,25 @@ let bottomCount = 0;
 
 headDropdown.addEventListener('change', async() => {
     // increment the correct count in state
-
+    headCount++;
     // update the head in supabase with the correct data
-    refreshData();
+    const newHead = await updateHead(headEl.value);
+
+    refreshData(newHead);
```

Comment:
`refreshData()` does not take any arguments.  Look at where the function is declared on line 118.